File: SERVER/test/com/variant/server/test/util/EventExperienceFromDatabase.java
package com.variant.server.test.util;


public class EventExperienceFromDatabase {

	long id;
	long eventId;
	String testName;
	String experienceName;
	boolean isControl;
	    
    EventExperienceFromDatabase() {}
    
	public long getId() { return id; }
	public long getEventId() { return eventId; }
	public String getTestName() { return testName; }
	public String getExperienceName() { return experienceName; }
	public boolean isControl() { return isControl; }

}

File: SERVER/test/com/variant/server/test/util/EventExperienceFromDatabase.scala
package com.variant.server.test.util;

import play.api.libs.json._

object EventExperienceFromDatabase {
   
   /**
    * Implicit writes converter from EventExperienceFromDatabase to JsValue
    */
   implicit val writes = new Writes[EventExperienceFromDatabase] {
     
      def writes(exp: EventExperienceFromDatabase) = 
         Json.obj(
            "eventId" -> exp.eventId,
            "testName" -> exp.testName,
            "expName" -> exp.experienceName,
            "isControl" -> exp.isControl
         )
}
}
class EventExperienceFromDatabase (
		val eventId: String,
		val testName: String,
		val experienceName: String,
		val isControl: Boolean
		) {
   
   
  
}
